released/packages/coq-paco/coq-paco.4.2.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "minki.cho@sf.snu.ac.kr"
+synopsis: "Coq library implementing parameterized coinduction"
+homepage: "https://github.com/snu-sf/paco/"
+dev-repo: "git+https://github.com/snu-sf/paco.git"
+bug-reports: "https://github.com/snu-sf/paco/issues/"
+authors: [
+  "Chung-Kil Hur <gil.hur@sf.snu.ac.kr>"
+  "Georg Neis <neis@mpi-sws.org>"
+  "Derek Dreyer <dreyer@mpi-sws.org>"
+  "Viktor Vafeiadis <viktor@mpi-sws.org>"
+  "Minki Cho <minki.cho@sf.snu.ac.kr>"
+]
+license: "BSD-3-Clause"
+build: [make "-C" "src" "all" "-j%{jobs}%"]
+install: [make "-C" "src" "-f" "Makefile.coq" "install"]
+depends: [
+  "coq" {>= "8.13" & < "8.17~"}
```

Comment:
```suggestion
  "coq" {>= "8.13" & < "8.18~"}
```
Coq 8.17.0 is just about to be released, and I tested locally that the package works with 8.17+rc1 (meaning that the release manager will ensure it keeps working with 8.17.0). So let's permit 8.17 in order to minimize future changes.